rfcs/intro.md

```
@@ -3,10 +3,31 @@
 ---
 
 
-# hopr-rfc
+# HOPR RFC Intro
 
 Request for Comments (RFC) for HOPR protocol
 
+# Table of Contents
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)